test/rekt/broker_test.go

```diff
@@ -64,42 +62,17 @@ func TestBrokerWithDLQ(t *testing.T) {
 		environment.Managed(t),
 	)
 
-	// The following will reuse the same environment for two different tests.
-
-	// Test that a Broker "test1" works as expected with the following topology:
-	// source ---> broker<Via> --[trigger]--> bad uri
+	// Test that a Broker works as expected with the following topology:
+	// source ---> broker --[trigger]--> bad uri
 	//                |
 	//                +--[DLQ]--> sink
-	// Setup data plane
-
-	// Install probes.
-	prober := eventshub.NewProber()
-	brokerName := "normal-broker"
-	sink1 := "sink"
-
-	// Wait till broker is ready since we need it to run the test
-	env.Prerequisite(ctx, t, broker.GoesReadyWithProbeReceiver(brokerName, sink1, prober, b.WithEnvConfig()...))
-	env.Test(ctx, t, broker.SourceToSinkWithDLQ(brokerName, sink1, prober))
-
-	// Test that a Broker "test2" works as expected with the following topology:
-	// source ---> broker +--[trigger<via1>]--> bad uri
-	//                |   |
-	//                |   +--[trigger<vai2>]--> sink2
-	//                |
-	//                +--[DLQ]--> sink1
-	// Wait till broker is ready since we need it to run this test
-	brokerName = "dls-broker"
-	sink2 := "sink2"
-	env.Prerequisite(ctx, t, broker.GoesReadyWithProbeReceiver(brokerName, sink1, prober, b.WithEnvConfig()...))
-	env.Test(ctx, t, broker.SourceToTwoSinksWithDLQ(brokerName, sink1, sink2, prober))
+	env.Test(ctx, t, broker.SourceToSinkWithDLQ())
 }
 
 // TestBrokerWithFlakyDLQ
 func TestBrokerWithFlakyDLQ(t *testing.T) {
 	t.Skip("Eventshub needs work")
 
-	class := eventing.MTChannelBrokerClassValue
-
```

Comment:
Since we're skipping this test, do we need changes in this test in this PR?

Reply:
True, let's test if this is not flaky first, and I'll remove those